degov.yml

```diff
@@ -0,0 +1,53 @@
+name: DeGov Development Test DAO
+logo: "/example/logo.svg"
+description: |
+  DeGov is a open-sourced governance platform that allows users to create
```

Comment:
`This is the development test DAO interface for DeGov users, allowing them to gain firsthand experience creating proposals and voting on them before deploying their own DeGov instance. The DAO is built on the Darwinia chain, an Ethereum-compatible chain.`